features/desktop/invitations.feature

```diff
@@ -49,6 +49,15 @@ Feature: Invitations
     Then I should have 1 Devise email delivery
     And I should not see "change your notification settings" in the last sent email
 
+  Scenario: sends an invitation from a page link
+    When I sign in as "alice@alice.alice"
+    And I click on selector "a.invitations-link"
```

Comment:
I'd prefer `And I click on "invite" within "#no_contacts"`. In the sidebar we also have the `invitations-link` class but in that case it is a `div`. This could be confusing.